lib/logstash/inputs/elasticsearch.rb

```diff
@@ -298,7 +301,12 @@ def process_next_scroll(output_queue, scroll_id)
   end
 
   def push_hit(hit, output_queue)
-    event = LogStash::Event.new(hit['_source'])
+    if @target.nil?
+      event = LogStash::Event.new(hit['_source'])
+    else
+      event = LogStash::Event.new
```

Comment:
I am assuming we deliberately do not want to treat the `'@timestamp': ...` field specially?
`LogStash::Event.new('@timestamp' => hit['_source']['@timestamp'])`

Reply:
I think this needs to be an either/or situation. Either we treat the `_source` _as_ the event (including magical fields) or we target the `_source` _to_ a place on the event (and the user then uses the `date` filter to pull the `@timestamp` over). If we attempt to mix, we will invariably get it wrong for some uses.

Reply:
OK, I am fine either way. Just more 'natural' for me to re-target regular fields (but not `@timestamp`).
Reading the same document and having different timestamps is surprising behavior despite the `target => ...` setting esp. since I can not have a nil `@timestamp` on the new event. Likely very subjective on my end ...

Reply:
resolving this by leaving as-is. `target` is about creating an event _with_ the `_source` of a hit, and without `target` is the path to create an event _from_ the `_source`.